<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_api_key">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_api_secret">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>480</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_api_secret">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>API SECRET</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>900</width>
     <height>130</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_api_key">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>API KEY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_login">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>330</y>
     <width>200</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
